core/storage/exploration/gae_models.py

```diff
@@ -865,6 +870,134 @@ def export_data(cls, user_id: str) -> Dict[str, List[str]]:
         }
 
 
+class TransientCheckpointUrlModel(base_models.BaseModel):
+    """Model for storing the progress of a logged-out user."""
+
+    # The exploration id.
+    exploration_id = (
+        datastore_services.StringProperty(required=True, indexed=True))
+    # The 6 digit unique id assigned to progress made by a logged-out user.
+    unique_progress_url_id = (
+        datastore_services.StringProperty(required=True))
+    # The state name of the furthest reached checkpoint.
+    furthest_reached_checkpoint_state_name = datastore_services.StringProperty(
+        default=None)
+    # The exploration version of the furthest reached checkpoint.
+    furthest_reached_checkpoint_exp_version = (
+        datastore_services.IntegerProperty(default=None))
+    # The state name of the most recently reached checkpoint.
+    most_recently_reached_checkpoint_state_name = (
+        datastore_services.StringProperty(default=None))
+    # The exploration version of the most recently reached checkpoint.
+    most_recently_reached_checkpoint_exp_version = (
+        datastore_services.IntegerProperty(default=None))
+    # Timestamp to store the creation time of the model.
+    creation_timestamp = (
+       datastore_services.DateTimeProperty(default=None))
+
+    @staticmethod
+    def get_deletion_policy() -> base_models.DELETION_POLICY:
+        """Model doesn't contain any data directly corresponding to a user."""
+        return base_models.DELETION_POLICY.NOT_APPLICABLE
+
+    @staticmethod
+    def get_model_association_to_user(
+    ) -> base_models.MODEL_ASSOCIATION_TO_USER:
+        """Model does not contain user data."""
+        return base_models.MODEL_ASSOCIATION_TO_USER.NOT_CORRESPONDING_TO_USER
+
+    @classmethod
+    def get_export_policy(cls) -> Dict[str, base_models.EXPORT_POLICY]:
+        """Model contains data to export corresponding to a logged-out user."""
+        return dict(super(cls, cls).get_export_policy(), **{
+            'exploration_id':
+                base_models.EXPORT_POLICY.NOT_APPLICABLE,
```

Comment:
Will it be `base_models.EXPORT_POLICY.NOT_APPLICABLE` if model contains data to be exported as mentioned in the docstring ?

Reply:
Fixed.